docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,606 @@
+{
```

Comment:
(NOTE: I forgot to provide the limits on summation. I will fix it)

The factor `-2/(sigma_square)` is constant, i.e. not depending on `t`.  So, we can take it out and  simply remove it. Now, if you expand the term by multiplying `X_t`, you can see we have: `SUM(TX - \mu X)`, which can be written as `SUM(TX) - SUM(\mu X)`. The latter term, `SUM(\mu X)` is zero because: `SUM(\mu X) = \mu SUM(X) = using eq(3) = 0` 

Please note that all `\mu` and all `\sigma` are constant. They might be known or unknown, but they are not changing with time stamp `t`. 

---

Well, now I see I should have provided the steps. When explanation goes beyond half a sentence, it should be a red flag for me that something is not written in a clear way. I can provide the steps that I skipped before. If you think that is not necessary, please let me know.

Reply:
I will revise it accordingly in the next set of commits.